scripts/build_locally.py

```diff
@@ -104,6 +105,16 @@ def run(
         # Always builds using oneMKL interfaces for the cuda target
         onemkl_interfaces = True
 
+    if target == "hip":
```

Comment:
What is about documentation updates?
We need to describe and provide an example how to enable build for HIP target.

Reply:
done